openjdk11_j9_special.system_ppc64le_linux Process DLT ended with exit code (143) no diagnostics

https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_special.system_ppc64le_linux_Nightly/2
DaaLoadTest_all_special_19
JVM_OPTIONS: -Xcompressedrefs -Xcompressedrefs -Xint -Xgcpolicy:balanced
```
10:43:27  DLT 14:43:26.530 - Completed 83.4%. Number of tests started=6002 (+35)
10:43:49  DLT 14:43:46.582 - Completed 83.4%. Number of tests started=6004 (+2)
10:44:07  DLT 14:44:06.512 - Completed 83.5%. Number of tests started=6012 (+8)
10:44:29  DLT 14:44:26.510 - Completed 83.6%. Number of tests started=6018 (+6)
10:44:47  DLT 14:44:46.531 - Completed 83.8%. Number of tests started=6031 (+13)
Cancelling nested steps due to timeout
10:44:49  Sending interrupt signal to process
10:44:51  STF 14:44:50.384 - **FAILED** Process DLT ended with exit code (143) and not the expected exit code/s (0)
10:44:51  STF 14:44:50.384 - Monitoring Report Summary:
10:44:51  STF 14:44:50.384 -   o Process DLT ended with exit code (143) and not the expected exit code/s (0)
10:44:51  STF 14:44:50.385 - Killing processes: DLT
10:44:51  STF 14:44:50.385 -   o Process DLT is not running
10:44:51  **FAILED** at step 1 (Run daa load test). Expected return value=0 Actual=1 at /home/jenkins/jenkins-agent/workspace/Test_openjdk11_j9_special.system_ppc64le_linux_Nightly/openjdk-tests/TestConfig/scripts/testKitGen/../../../TestConfig/test_output_15642029557738/DaaLoadTest_all_special_19/20190727-110317-DaaLoadTest/execute.pl line 96.
10:44:51  STF 14:44:50.848 - **FAILED** execute script failed. Expected return value=0 Actual=1
10:44:51  STF 14:44:50.848 - 
10:44:51  STF 14:44:50.849 - ====================   T E A R D O W N   ====================
10:44:51  STF 14:44:50.849 - Running teardown: perl /home/jenkins/jenkins-agent/workspace/Test_openjdk11_j9_special.system_ppc64le_linux_Nightly/openjdk-tests/TestConfig/scripts/testKitGen/../../../TestConfig/test_output_15642029557738/DaaLoadTest_all_special_19/20190727-110317-DaaLoadTest/tearDown.pl
10:44:51  Terminated
10:44:51  
10:44:51  DaaLoadTest_all_special_19_FAILED
10:44:51  
10:44:51  "/home/jenkins/jenkins-agent/workspace/Test_openjdk11_j9_special.system_ppc64le_linux_Nightly/openjdkbinary/j2sdk-image/bin/java" -Xshareclasses:destroyAll; "/home/jenkins/jenkins-agent/workspace/Test_openjdk11_j9_special.system_ppc64le_linux_Nightly/openjdkbinary/j2sdk-image/bin/java" -Xshareclasses:groupAccess,destroyAll; echo "cache cleanup done";
10:44:51  autoGen.mk:4211: recipe for target 'DaaLoadTest_all_special_19' failed
10:44:51  make[2]: *** [DaaLoadTest_all_special_19] Terminated
10:44:51  Terminated
10:44:51  /home/jenkins/jenkins-agent/workspace/Test_openjdk11_j9_special.system_ppc64le_linux_Nightly/openjdk-tests/TestConfig/scripts/testKitGen/../../../TestConfig/settings.mk:281: recipe for target 'special.system-daaLoadTest' failed
10:44:51  make[1]: *** [special.system-daaLoadTest] Error 143
10:44:51  make[1]: Leaving directory '/home/jenkins/jenkins-agent/workspace/Test_openjdk11_j9_special.system_ppc64le_linux_Nightly/openjdk-tests/systemtest'
10:44:51  /home/jenkins/jenkins-agent/workspace/Test_openjdk11_j9_special.system_ppc64le_linux_Nightly/openjdk-tests/TestConfig/scripts/testKitGen/../../../TestConfig/settings.mk:281: recipe for target 'special.system-systemtest' failed
10:44:51  make: *** [special.system-systemtest] Error 2
10:44:51  make: Leaving directory '/home/jenkins/jenkins-agent/workspace/Test_openjdk11_j9_special.system_ppc64le_linux_Nightly/openjdk-tests'
10:44:51  
10:44:51  Attempting to destroy all caches in cacheDir /home/jenkins/javasharedresources/
10:44:51  
10:44:51  JVMSHRC806I Compressed references persistent shared cache "sharedcc_jenkins" has been destroyed. Use option -Xnocompressedrefs if you want to destroy a non-compressed references cache.
10:44:51  script returned exit code 2
[Pipeline] sh
10:44:52  + uname
10:44:52  + [ Linux = AIX ]
10:44:52  + MAKE=make
10:44:52  + make -f ./openjdk-tests/TestConfig/testEnv.mk testEnvTeardown
10:44:52  make: Nothing to be done for 'testEnvTeardown'.
[Pipeline] }
[Pipeline] // timestamps
[Pipeline] }
[Pipeline] // stage
[Pipeline] cleanWs
[WS-CLEANUP] Deleting project workspace...
[WS-CLEANUP] Deferred wipeout is used...
[WS-CLEANUP] done
[Pipeline] }
[Pipeline] // timeout
[Pipeline] }
[Pipeline] // node
[Pipeline] }
[Pipeline] // stage
[Pipeline] End of Pipeline
Timeout has been exceeded
```

--- a/buildenv/jenkins/variables/defaults.yml
+++ b/buildenv/jenkins/variables/defaults.yml
@@ -142,6 +142,8 @@ ppc64le_linux:
       13: 'CC=gcc-7 CXX=g++-7'
       next: 'CC=gcc-7 CXX=g++-7'
   excluded_tests:
+    11:
+      ? special.system
     12:
       ? special.system
     13:
@@ -297,8 +299,6 @@ x86-64_linux:
       13: 'source /opt/rh/devtoolset-7/enable'
       next: 'source /opt/rh/devtoolset-7/enable'
   excluded_tests:
-    8:
-      ? special.system
     11:
       ? special.system
     12:

